docs/api/apiv3/example/README.md

```diff
@@ -90,23 +90,35 @@ For this guide, we again chose Postman as our client application. Apart from "Ba
 
 ![OAuth2 postman configure](./oauth2-postman-configure.png)
 
-There are a couple of fields in that form but most of the information can simply be copied over:
+Please use the following instructions to set these fields for you:
 
-* Token Name: Freely choosable field
+* Token Name: Freely choosable field, 
+for example "Token Name"
 * Grant Type: Choose "Authorization Code"
-* Callback URL: Copy from OpenProject
-* Auth URL: Copy from OpenProject
-* Access Token URL: Copy from OpenProject
-* Client ID: Copy from OpenProject
-* Client Secret: Copy from OpenProject
-* Scope: Type in "api_v3" as we chose that in OpenProject
+* **Callback URL**: This is the callback URL of the _client_ _application_.
+With Postman 9.6 this is "https://oauth.pstmn.io/v1/callback".
+Postman automatically fills this field with this URL
+when you select "Authorize using browser". However, you have to
+uncheck "Authorize using browser" afterwards again, please see below.
```

Comment:
Not sure I follow, I never unchecked this flag but the token was still verified. It should work in both cases. If you leave it checked, it will simply reuse your browser's session instead of opening a "browser" window within postman